sdk/src/main/java/co/omise/android/ui/CreditCardEditText.kt

```diff
@@ -85,12 +85,12 @@ class CreditCardEditText : OmiseEditText {
         cardBrandImagePaint = Paint(Paint.ANTI_ALIAS_FLAG)
     }
 
-    override fun onDraw(canvas: Canvas?) {
+    override fun onDraw(canvas: Canvas) {
```

Comment:
The `nullable` method is not available anymore in api level 34

Reply:
Any idea why was it was initially designed as nullable? Will this change impacts our system negatively?

Reply:
The nullibility of a parameter may change due to many reasons but some of them are to have enhanced design patterns or to have a better usage of kotlin null safe features. 

The sdk is compiled separately from the settings of the host app so it should not affect the users. I have already tried it with the lowest allowed api on Google playstore (31) and it works fine.